File: 3_isPrime/isPrime.py
import math

def isPrime(n):
	for x in range(2, int(math.sqrt(n)+1)):
		if (n % x == 0):
			return False
	return True

def main():
    n = input("Please enter a number: ")
    print(isPrime(float(n)))

if __name__ == "__main__":
    main()

File: 3_isPrime/isPrime.c
#include <stdio.h>
#include <math.h>

int isPrime(int n){
  int i;
  for (i = 2; i <= floor(sqrt(n)); i++){
    if (n % i == 0)
      return 0;
  }
  return 1;
}

int main(){
  int n;
  printf("Please enter a number: ");
  scanf("%d", &n);
  int res = isPrime(n);
  if (res){
    printf("True");
  }else{
    printf("False");
  }
}
